language/command-line-common/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "command-line-common"
```

Comment:
Given that we may want to publish our crates to crates.io in the future, I would suggest we use a prefixed name like "move-command-line-common"

Reply:
Sure, why not